### PR TITLE
Content types

### PR DESCRIPTION
**Pull request by MattTennison**
# What?

Add content type to MUD Vocab

# Why?

To support changes in https://github.com/Multi-User-Domain/mud-jena/pull/7

### CODE REVIEW

File: mud.ttl [hunk 1]
```diff
@@ -102,3 +102,12 @@ mud:CharacterList a rdf:Property ;
...
     rdfs:range xsd:string .
+
+mud:content a rdfs:Class ;
+    rdfs:label "Textual Content"@en ;
+    rdfs:comment "Content describing another model"@en ;
+
+mud:sight a rdfs:Property
+    rdfs:label "Sight"@en ;
+    rdfs:comment "The visual sense"@en ;
+    rdfs:domain mud:content
```
Comment (calummackervoy) [inline mud.ttl]: Could we move the new things into a new ontology please ? I'm thinking `mudcontent` ?

Just for modularity - if someone was writing their own front-end they might want to replace our definitions for the user's perception

**Review by calummackervoy — CHANGES_REQUESTED** [review]

File: mud.ttl (hunk 1 above)
Comment (MattTennison) [reply mud.ttl]: Done in https://github.com/Multi-User-Domain/vocab/pull/2/commits/1a6f2861f31eea0fef6de765ad814c5708eec396 :smile: 

I then updated the Java app here https://github.com/Multi-User-Domain/mud-jena/pull/7/commits/59a821aff2a2418195fdb902bd7d6dd5b9094885

File: mudcontent.ttl [hunk 2]
```diff
@@ -0,0 +1,25 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
```
Comment (calummackervoy) [inline mudcontent.ttl:2]: I added some extra imports. Note that whenever you use the syntax `mud:something` it's actually shorthand for the full URI https://raw.githubusercontent.com/Multi-User-Domain/vocab/main/mud.ttl#something

The prefix statements define what the shorthand is for

File: mudcontent.ttl [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    dcterms:creator <https://matttennison.inrupt.net/profile/card#me> .
+
+mudcontent:content a rdfs:Class ;
+  rdfs:label "Textual Content"@en ;
+  rdfs:comment "Content describing another model"@en ;
+
+mudcontent:sight a rdfs:Property
+  rdfs:label "Sight"@en ;
+  rdfs:comment "The visual sense"@en ;
+  rdfs:domain mud:content .
```
Comment (calummackervoy) [inline mudcontent.ttl:25]: added a missing '.' character here. Every turtle statement ends in a full stop, the semi-colon is a break to say "done with this property, but the next one is about the same subject (`mudcontent:sight`)

**Review by calummackervoy — APPROVED** [review]
congrats on publishing your first ontology :wink:

**Pull request merged**